scripts/bootstrap-aio.yml

```diff
@@ -158,8 +158,8 @@
   include: "{{ lookup('env', 'OA_DIR') }}/tests/bootstrap-aio.yml"
   vars:
     bootstrap_host_apt_distribution_suffix_list: "{{ (lookup('env', 'RPCO_APT_ARTIFACTS_AVAILABLE') | bool) | ternary([], ['updates', 'backports']) }}"
-    bootstrap_host_scenario: "{% if lookup('env', 'DEPLOY_MAGNUM') == 'yes' %}magnum{% elif lookup('env', 'DEPLOY_IRONIC') == 'yes' %}ironic{% else %}{{ lookup('env', 'DEPLOY_CEPH') |bool |ternary('ceph','swift') }}{%endif%}"
```

Comment:
Why are we removing `DEPLOY_MAGNUM` in this commit?

Reply:
I thought we REMOVED magnum.

Reply:
Newton did

Reply:
Magnum is deprecated, but doesn't need to be removed in this commit.
Let's leave `DEPLOY_MAGNUM` alone and address removal in a separate PR.

Reply:
True, though if we don't do it here we should document it for removal later.

Reply:
addressed.

Reply:
Actually, ``DEPLOY_MAGNUM`` was added by https://github.com/rcbops/rpc-openstack/commit/e2fdc5989f236da5d3e1135b83bf0c4b7dc8c6b2 mistakenly, so it should be removed in this PR.

Reply:
https://github.com/rcbops/rpc-openstack/blob/newton/scripts/bootstrap-aio.yml

it is not in newton